daemon/src/main.rs

```diff
@@ -55,7 +55,11 @@ mod tray;
 mod tts;
 
 const VERSION: &str = env!("CARGO_PKG_VERSION");
+
+#[cfg(not(target_os = "macos"))]
 const ICON: &[u8] = include_bytes!("../resources/goxlr-utility-large.png");
+#[cfg(target_os = "macos")]
+const ICON: &[u8] = include_bytes!("../../ci/macos/icon.icns");
```

Comment:
Could you move this to a unique variable inside `macos.rs`, there may be a reason to use the regular PNG for something else, and I'd like for that to be available and generic for all platforms.